src/index/terrain-viewer.ts

```diff
@@ -32,14 +32,15 @@ export class TerrainViewer {
   constructor(doms: Doms) {
     this.doms = doms;
     this.texture = new three.CanvasTexture(doms.texture);
+    this.texture.colorSpace = three.SRGBColorSpace;
```

Comment:
Necessary parameters were not set from the first implementation.

https://threejs.org/docs/#api/en/textures/Texture.colorSpace

> Textures containing color data should be annotated with [THREE.SRGBColorSpace](https://threejs.org/docs/index.html#api/en/constants/Textures) or [THREE.LinearSRGBColorSpace](https://threejs.org/docs/index.html#api/en/constants/Textures).